layouts/partials/homepage/role_slider.html

```diff
@@ -1,11 +1,19 @@
-<div class="tabbed">
+<div id="roleSlider" class="tabbed bg-card-dark-background-color">
```

Comment:
I might ask that you go into the css file and just `@apply` the changes to the tabbed class instead of adding them here.

Reply:
addressed!